_docs/identity.md

```diff
@@ -1,73 +1,86 @@
 ---
-title: Set up Identity
+title: Setup an on-chain identity
 subtitle: This document covers the steps of setting up on-chain identity for your account
 author: charles
 tags: [identity, registrar]
 ---
 
-## Intro
+## How to setup your on-chain identity
 
-**For better collaboration, we strongly suggest each developer to set up an identity on Turing Network. Here is how.**
+### Step 1: Navigate to your accounts page on PolkadotJS
 
-## Set Identity
-
-On [polkadot.js apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/) page, select the “Accounts” tab and select “Set on-chain identity” with your account as shown below.
+On [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/) page, select the “Accounts” tab and select “Set on-chain identity” with your account as shown below.
 
 ![set-on-chain-identity](../../assets/img/identity/set-on-chain-identity.png)
 
+### Step 2: Select the account you want to set an identity for and fill in the inputs
+
 Enter your identity information and click the “Set Identity” button.
 
 ![register-identity](../../assets/img/identity/register-identity.png)
 
+### Step 3: Validate that your information is correct after saving your identity
+
 After the extrinsic is confirmed, there will be a gray minus sign to the left of the account name, indicating that the account has been registered but has not yet passed the judegement.
 
 ![no-judgements](../../assets/img/identity/no-judgements.png)
 
-## Request Judegement
+### Step 4: Request a judgement
 
-### Retrieving the List of Registrars
+Navigate to **Developer > Extrinsics**, and use your account to send a `identity.requestJudgement` extrinsic.
 
-![registrars](../../assets/img/identity/registrars.png)
+![request-judgement](../../assets/img/identity/request-judgement.png)
 
-As shown above, there are currently two registrars.
+- regIndex: The index of registrars.
+- maxFee: You need to provide a fee in `Planck` higher than registrar.
+
+### Step 5: Contact a registrar to acquire the judgement
 
-The fee of the first registrar (6Ah9LrGFyc4AT22dYUQTuSY8vxJkegZ5F5Udqx4eVJh8AJ6T) is 0, which means that he charges 0 tokens to do a judgement.
+You can either contact the registrar via their on-chain information or you can ask the registrar to review your request in [OAK Discord room](https://discord.gg/7W9UDvsbwh).
```

Comment:
It would be good to specify a discord channel for folks to send their registrar requests to so that we can monitor it if that's a concern.